app/src/main/java/com/safetyheads/akademiaandroida/YouTube/useCases/ChannelUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.safetyheads.akademiaandroida.YouTube.useCases
+
+import com.safetyheads.akademiaandroida.YouTube.entities.chanel.ChannelDataClass
+import com.safetyheads.akademiaandroida.network.NetworkResult
+import kotlinx.coroutines.flow.Flow
+
+interface ChannelUseCase {
```

Comment:
In general we don't need `interface` for UseCases, our UseCases should implement only BaseUseCase